<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Procedural Maze Generation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainWindowGridLayout">
    <item row="8" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="3">
     <layout class="QHBoxLayout" name="colorHL">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Cube Color</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cubeColorCombo"/>
      </item>
     </layout>
    </item>
    <item row="8" column="3">
     <layout class="QHBoxLayout" name="selectAlgo_HL">
      <item>
       <widget class="QLabel" name="algoLabel">
        <property name="text">
         <string>Select Algorithm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="algoComboBox"/>
      </item>
     </layout>
    </item>
    <item row="17" column="3">
     <layout class="QHBoxLayout" name="bottomBtns_HL">
      <item>
       <widget class="QPushButton" name="generateMazeBtn">
        <property name="text">
         <string>Generate</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="resetMazeBtn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="3">
     <widget class="QTextBrowser" name="textBrowser">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Instructions&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="13" column="3">
     <layout class="QHBoxLayout" name="mazeSpecs_HL">
      <item>
       <widget class="QSpinBox" name="widthMaze">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Width</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="depthMaze">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="minimum">
         <number>15</number>
        </property>
        <property name="maximum">
         <number>70</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Depth</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="18" column="3">
     <widget class="QCheckBox" name="wireframe_checkBox">
      <property name="text">
       <string>Wireframe</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="3" column="3">
     <layout class="QHBoxLayout" name="FloorHL">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Floor Color     </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="floorColorCombo"/>
      </item>
     </layout>
    </item>
    <item row="4" column="3">
     <layout class="QHBoxLayout" name="cameraSwitch_HL">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Select Camera</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="topRadio">
        <property name="text">
         <string>TopDown</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="defaultRadio">
        <property name="text">
         <string>Default</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="3">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>A program to generate maze procedurally.</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Maze Generation Program</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
